### PR TITLE
Add the /edit switch to the @help command

### PR DESCRIPTION
**Pull request by vlegoff**
#### Brief overview of PR changes/additions

This just adds a /edit switch to the @help command, allowing to edit the text of the help entry in an EvEditor.

#### Motivation for adding to Evennia

It's easier for some users to manage long help entries within an editor, rather than handling it through the other switches.

### CODE REVIEW

File: evennia/commands/default/help.py [hunk 1]
```diff
@@ -228,14 +229,35 @@ def func(self):
         self.msg(self.format_help_entry("", "No help entry found for '%s'" % query, None, suggested=suggestions))
 
 
+def loadhelp(caller):
```
Comment (Griatch) [inline evennia/commands/default/help.py]: These should probably be made into private functions (starting with `_`); then they also don't need the full docstrings (which would not help readability in this case).

**Comment by Griatch** [comment]
I like the idea but this does not seem to work for me. Entering `@sethelp/edit foo` or `@sethelp/edit foo = testtext` both tell me that topic "foo" could not be found.

**Comment by vlegoff** [comment]
@Griatch: perfectly true, sorry.  The switch works when the topic exists, to edit its content, but not to create it.  I will update the code.

**Comment by Griatch** [comment]
Thanks! I made only some small further usability changes, like allowing for `@sethelp/edit topicname` without having to give the the right-hand-side text as input (and appending it to the buffer if you do). Merged with a rebase.

**Pull request closed without merging**